docs/05-themes-components/01-developing-themes.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Developing Discourse Themes & Theme Components
+short_title: Developing Themes
+id: beginners-guide
+---
+
+Discourse Themes and Theme Components can be used to customize the look, feel and functionality of Discourse's frontend. This section of the developer guides aims to provide all the reference materials you need to develop simple themes for a single site, right up to complex open-source theme components.
+
+This introduction aims to provide a map of all the tools and APIs for theme development. If you prefer a step-by-step tutorial for theme development, jump straight to out [Building your first theme]().
```

Comment:
> [Building your first theme]().

are we missing  a link here?

Reply:
Yup, it'll link to the first step of the tutorial. We need to sync to meta first, then I'll update all the missing links in a follow-up PR.

(not ideal... but that's the best we can do with the current minimum-viable syncing system)